### DISCUSSION TITLE
Be Junior

### DISCUSSION THREAD

**Discussion by zupersponge** [iptv-org/community#678] [Show and tell]
**Channel Name:** beJunior
**Channel ID:** BeJunior.qa
**Country:** (Qatar?)
**Language:** Arabic & English
**External Link:** 
**Notes (optional):** Not found a link yet, I will respond to this thread if I find one. (if someone else has also requested this channel, you may close my thread)

**Comment by BellezaEmporium** [comment]
I confirm it's a Qatari channel. beJunior is linked to the beIN Media Group channels.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19676

**Reply by Brunisky14** [reply]
m3u8 link?
